packages/widgets/src/blocks/widget-group/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+import { group as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+// eslint-disable-next-line no-unused-vars
```

Comment:
I think you forgot to delete this 😅.

Reply:
Consider it gone. 6bf3b0ec7031732a2455a28f495fcab788271292.